internal/records/events.go

```diff
@@ -156,7 +156,16 @@ func ReconcileEvents(ctx context.Context, now time.Time, ups map[string]Upness,
 
 	for key, up := range ups {
 		rec := events[key]
-		reconcileLog.Info("ReconcileEvents", "key", key, "expected", up.ExpectedCount, "ready", up.ReadyCount, "unknownCount", up.UnknownCount, "unknownThreshold", unknownThreshold)
+		var keyName string
+
+		if up.Attrs.NodePoolName != "" {
+			keyName = "nodepool_name"
+		} else if up.Attrs.JobSetUID != "" {
+			keyName = "jobset_uid"
+		} else {
+			keyName = "key"
+		}
+		reconcileLog.Info("ReconcileEvents", keyName, key, "expected", up.ExpectedCount, "ready", up.ReadyCount, "unknownCount", up.UnknownCount, "unknownThreshold", unknownThreshold)
```

Comment:
There is already a "type" key, to determine if it's jobsets, jobset-nodes or nodepools. Let's remove this from this change unless it's required